content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+   ```shellsession
+   GET /backup-recovery/v1beta1/protection-stores?select=name,displayName,id,status,state,protectionStoreType&filter=protectionStoreType eq ‘ON_PREMISES’ and storageSystemInfo/id eq “\<protection-store-gateway-id\>”
+   ```
+
+![API to obtain the onpremises protection store id](/img/api-to-get-onpremises-protection-store-id.png)
+
+4. I repeat the same execution of the `GET /backup-recovery/v1beta1/protection-stores` to obtain the **“<cloud-protection-store-id>”**. To accomplish that, I used the following `filter: protectionStoreType eq 'CLOUD' and storageSystemInfo/id eq “<protection-store-gateway-id>”`.  Additionally, I also used the  parameter `select: name,displayName,id,status,state,protectionStoreType` to provide shorter response for simpler discovery of the protection-store-id in the cloud. The API used for this: 
+
+   ```shellsession
+   GET /backup-recovery/v1beta1/protection-stores?select=name,displayName,id,status,state,protectionStoreType&filter=protectionStoreType eq ‘CLOUD’ and storageSystemInfo/id eq “\<protection-store-gateway-id\>.”
+   ```
+
+![API to obtain cloud protection store id](/img/api-discover-cloud-protection-store-id.png)
+
+5. For the next step, I created a request body JSON structure that represents the protection policy schedule and each of the protection stores. Inside this JSON structures for request body, I defined the three objects that represent the `SNAPSHOT, BACKUP (on-premises), CLOUD_BACKUP`. Note that this structure can be expanded or contracted depending on the required backup strategy. The SNAPSHOT object did not require `"<protection-store-Id>"` as that recovery points will exist inside the primary storage array. This request JSON body structure was required to create the protection policy using HPE GreenLake [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/DataManagementTemplateCreate/) `POST /backup-recovery/v1beta1/protection-policies`.
+
+> ***Note:*** I didn’t include objects for immutability, prescript, and postscript into the JSON structure. If it’s not intended, you don’t need to include unused key-pair values into the JSON structure. Additionally, the SNAPSHOT object does no require a `protectionStoreId`.
```

Comment:
Do you always italicize your notes? I don't remember it that way. However you do it, be consistent.